snapcraft/plugins/rust.py

```diff
@@ -97,11 +97,24 @@ def __init__(self, name, options, project):
         self._rustup = os.path.join(self._rustpath, "rustup.sh")
         self._manifest = collections.OrderedDict()
 
+    def _test(self):
+        if self.project.is_cross_compiling:
```

Comment:
it would be nice to issue a warning here about skipping the test execution since it is cross compiling.